src/axom/inlet/Table.hpp

```diff
@@ -16,6 +16,8 @@
 
 #include <memory>
 #include <string>
+#include <functional>
+#include <unordered_set>
```

Comment:
It looks like only `unordered_map` is used in this header and not `unordered_set`